magento/magento2
# Fixed typo in unit test introduced by accident in e0c61dee13afcd10c03…

…3803e344a22df10631eeb

### Description (*)
Accidentally found by running phpstan on level 0 on the entire codebase. `$this->attribute§('getIsUnique')` is not really proper code.

It looks like this mistake was accidentally introduced in https://github.com/magento/magento2/commit/e0c61dee13afcd10c033803e344a22df10631eeb#diff-220a7a1cb2d4bae1236aeeb318f8ae5368fb92017598c1db77fba6754d2cf9fa

I've also marked the test to no longer being skipped, it seems to run without any issues after this fix. I have no idea what is being mentioned in MAGETWO-48956?

### Related Pull Requests

### Fixed Issues (if relevant)

### Manual testing scenarios (*)
1. Run `vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist app/code/Magento/Catalog/Test/Unit/Model/Category/Attribute/Backend/SortbyTest.php` and make sure it doesn't output errors

### Questions or comments

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#37792: Fixed typo in unit test introduced by accident in e0c61dee13afcd10c03…
